exercises/practice/high-scores/.meta/example.sql

```diff
@@ -0,0 +1,69 @@
+-- store comma-separated list of scores of a UUID
+UPDATE results
+SET result = score_groups.scores
+FROM (
+    SELECT uuid, GROUP_CONCAT(score) "scores"
+    FROM (
+        SELECT scores.uuid, score
+        FROM scores
+        INNER JOIN results USING (uuid)
+        WHERE results.property = 'scores'
+        ORDER BY scores.rowid
+    ) scores_by_uuid
+    GROUP BY uuid
+) score_groups
+WHERE results.uuid = score_groups.uuid;
+
+-- store the latest score by UUID
+UPDATE results
+SET result = latest_score.score
+FROM (
+    SELECT scores.uuid, score
+    FROM scores
+    INNER JOIN (
+        SELECT scores.uuid, MAX(scores.rowid) "last_row"
+        FROM scores
+        INNER JOIN results USING (uuid)
+        WHERE results.property = 'latest'
```

Comment:
It might be worth documenting in an `append.md` what the `results.property` values can be and how they need to be used. Without understanding that field, the expectations here are a bit confusing.